index.html

```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <script defer src = index.js></script>
-    <title>Document</title>
-</head>
-<body id="body">
-    
-</body>
-</html>
\ No newline at end of file
+    <head>
```

Comment:
it's totally common to have the head and body elements on the same indented level as the html elements, even though they're children of html, and should therefore be indented. this makes no sense to me. religious devotion to indenting is a virtue and it starts at the first element.